dwave/cloud/cli.py

```diff
@@ -1051,6 +1051,32 @@ def refresh(*, config_file, profile, output):
            'You can now use "dwave auth get" to view them.')
 
 
+@auth.command()
+@config_file_options()
+@click.argument('token_type', default='access-token',
+                type=click.Choice(['access-token', 'refresh-token']))
+@json_output
+@standardized_output
+def revoke(*, config_file, profile, token_type, json_output, output):
+    """Revoke Leap API access and/or refresh token(s)."""
```

Comment:
```suggestion
    """Revoke access and/or refresh token(s) for Leap API."""
```

Reading "Revoke Leap API access" will scare people